client/src/App.js

```diff
@@ -38,6 +39,14 @@ class App extends Component {
     newComposition.push(clickedMelody);
   }
 
+  handleRobotPlayToggled(i, play) {
+    console.log(`Toggled play to ${play} for robot ${i}`);
+    const bot = { ...this.state.bots[i], metric: 0 + play };
```

Comment:
Thanks for recording the number of plays!
`metric` is used for favouriting right now. Maybe we can add another field, `plays`, or make the `metric` field into an object with `plays`, `favourite`, and any other fields we choose to include in the future.